src/soko_game.rs

```diff
@@ -42,10 +44,18 @@ pub fn handle_key(key: event::KeyEvent) -> Option<GameAction> {
         KeyCode::Char('r') => Some(GameAction::Reset),
 
         // Movement
-        KeyCode::Up | KeyCode::Char('w') => Some(GameAction::Move(Direction::Up)),
-        KeyCode::Left | KeyCode::Char('a') => Some(GameAction::Move(Direction::Left)),
-        KeyCode::Down | KeyCode::Char('s') => Some(GameAction::Move(Direction::Down)),
-        KeyCode::Right | KeyCode::Char('d') => Some(GameAction::Move(Direction::Right)),
+        KeyCode::Up | KeyCode::Char('w') | KeyCode::Char('W') => {
```

Comment:
Why caps?

Reply:
why do you want the game to be unplayable when the user has caps enabled, I would ask :)